cypress_test/integration_tests/desktop/calc/bottom_bar_spec.js

```diff
@@ -12,8 +12,14 @@ describe(['tagdesktop', 'tagnextcloud', 'tagproxy'], 'Calc bottom bar tests.', f
 	it('Bottom tool bar.', function() {
 		cy.cGet('#map').focus();
 		calcHelper.clickOnFirstCell();
-		cy.cGet('#StateTableCellMenu').click();
-		// If it clicks, it passes.
-		cy.cGet('body').contains('.ui-combobox-entry', 'CountA').click();
+		cy.cGet('#StateTableCellMenu').then(($tableCellMenu) => {
+			if ($tableCellMenu.is(':visible')) {
```

Comment:
Please avoid conditional checks in cypress, it makes it unstable.
Please to explicit checks for visible element and explicit check if something is hidden or missing.
`cy.cGet('#StateTableCellMenu:visible').click();` <- does what that code does. If it will fail - cypress will print that it was invisible.

We want strict checks, which can be repeated automatically.